Vision/Vision.bs

```diff
@@ -158,6 +158,9 @@ Markup Shorthands: markdown yes
 	<li id=free-impl>
 		**Free to Implement**: Our standards are rooted in a strong royalty-free patent policy
 		and open copyright licenses.
+	<li id=voluntary-impl>
+		**Voluntary to Implement**: Our standards span many use-cases and diverse industries, 
+                are thus developed to be voluntarily adopted, and success is determined by the market.
```

Comment:
I'm not following the logic of the "and thus", it feels like a non-sequitur... maybe you can explain a bit better?